src/platform/web/ui/session/room/timeline/BaseMessageView.js

```diff
@@ -120,6 +120,7 @@ export class BaseMessageView extends TemplateView {
         } else if (vm.canRedact) {
             options.push(Menu.option(vm.i18n`Delete`, () => vm.redact()).setDestructive());
         }
+        options.push(Menu.option(vm.i18n`Copy permalink`, () => vm.copyPermalink()));
```

Comment:
Should it say `matrix.to` in it?

```suggestion
        options.push(Menu.option(vm.i18n`Copy matrix.to permalink`, () => vm.copyPermalink()));
```

Reply:
I reckon it should say "Copy `matrix.to` permalink". Updating to this ✅